pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>com.baidu.hugegraph</groupId>
     <artifactId>hugegraph-client</artifactId>
-    <version>1.9.3</version>
+    <version>1.9.4</version>
```

Comment:
update travis server_commit_id in start-hugegraph-server.sh

Reply:
updated it in `.travis.yml` & CI passed